### PR TITLE
how-to/get-started: add a note

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Add a note
### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/issues/1682
### Which version does your change affect? 

dev, v2.1, v3.0, v3.1

### CODE REVIEW

**Comment by CaitinChen** [comment]
@imtbkcat PTAL

File: dev/how-to/get-started/import-example-database.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ end_station_number, end_station, bike_number, member_type);
 
 ### Import all files
 
+**Note:**
+
+For the MySQL client, the local data loading feature is disabled by default. Before you perform the following step, make sure the MySQL client has enabled `local-infile`.
```
Comment (imtbkcat) [inline dev/how-to/get-started/import-example-database.md]:
```suggestion
For the MySQL client, the local data loading local file feature is disabled by default. Before you perform the following step, make sure the MySQL client has enabled `local-infile`.
```

File: v2.1/how-to/get-started/import-example-database.md [hunk 2]
```diff
@@ -54,6 +54,10 @@ end_station_number, end_station, bike_number, member_type);
 
 ### Import all files
 
+**Note:**
+
+For the MySQL client, the local data loading feature is disabled by default. Before you perform the following step, make sure the MySQL client has enabled `local-infile`.
```
Comment (imtbkcat) [inline v2.1/how-to/get-started/import-example-database.md]:
```suggestion
For the MySQL client, the local data loading local file feature is disabled by default. Before you perform the following step, make sure the MySQL client has enabled `local-infile`.
```

File: v3.0/how-to/get-started/import-example-database.md [hunk 3]
```diff
@@ -55,6 +55,10 @@ end_station_number, end_station, bike_number, member_type);
 
 ### Import all files
 
+**Note:**
+
+For the MySQL client, the local data loading feature is disabled by default. Before you perform the following step, make sure the MySQL client has enabled `local-infile`.
```
Comment (imtbkcat) [inline v3.0/how-to/get-started/import-example-database.md]:
```suggestion
For the MySQL client, the local data loading local file feature is disabled by default. Before you perform the following step, make sure the MySQL client has enabled `local-infile`.
```

**Comment by jackysp** [comment]
I think we've already had an FAQ, https://pingcap.com/docs-cn/stable/faq/tidb/#923-error-1148-42000-the-used-command-is-not-allowed-with-this-tidb-version-%E9%97%AE%E9%A2%98%E7%9A%84%E5%A4%84%E7%90%86%E6%96%B9%E6%B3%95

**Comment by CaitinChen** [comment]
> I think we've already had an FAQ, https://pingcap.com/docs-cn/stable/faq/tidb/#923-error-1148-42000-the-used-command-is-not-allowed-with-this-tidb-version-%E9%97%AE%E9%A2%98%E7%9A%84%E5%A4%84%E7%90%86%E6%96%B9%E6%B3%95

I think notifying readers in advance is better.

**Comment by CaitinChen** [comment]
@imtbkcat Please sign CLA.

**Comment by CaitinChen** [comment]
@jackysp @imtbkcat has updated commands in his last commit. PTAL.

**Pull request merged**